admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -302,4 +320,15 @@ export class AlertBannerModel extends HoistModel {
         await XH.alertBannerService.checkForBannerAsync();
         await this.refreshAsync();
     }
+
+    private async getRolesAsync() {
+        try {
+            const roles = await XH.fetchJson({
+                url: 'roleAdmin/list'
+            });
```

Comment:
This service might not always be enabled - some apps don't use Hoist's built-in role management.  In that case, this would throw.  Looks like we should have this call `roleAdmin/config` first to determine if enabled. If so, go on to load the roles. If not, might end up updating the UI, depending on how much we say about roles that's visible to users.